internal/slack/files.go

```diff
@@ -0,0 +1,203 @@
+package slack
```

Comment:
I agree, it's just proving to be a real pain to generate an appropriate testing artifact to process, which is what I'm working on / have been working on the last couple of days. The artifact I have been using for manual testing has some sensitive data in it, so I'm working on something to scrub the sensitive data from the archive so that it can be used in tests.

It's also possible that I might be able to refactor this file somewhat to make it more testable. It's borrowed from the slack-advanced-exporter which was, in my understanding, written in a bit of a hurry, so I just haven't taken the time to make it more easily testable or more modular.